src/main/java/com/semmle/jira/addon/workflow/LgtmTransitionNotificationFunction.java

```diff
@@ -35,24 +39,38 @@
   @ComponentImport private final PluginSettingsFactory pluginSettingsFactory;
   @ComponentImport private final TransactionTemplate transactionTemplate;
 
+  private final PluginSettings settings;
+
   @Inject
   LgtmTransitionNotificationFunction(
       PluginSettingsFactory pluginSettingsFactory, TransactionTemplate transactionTemplate) {
     this.pluginSettingsFactory = pluginSettingsFactory;
     this.transactionTemplate = transactionTemplate;
+    this.settings = pluginSettingsFactory.createGlobalSettings();
   }
 
+  @Override
   @SuppressWarnings("rawtypes")
   public void execute(Map transientVars, Map args, PropertySet ps) throws WorkflowException {
 
-    String configKey = "webhook"; // TODO get config key from custom field
+    Transition transition = getTransitionParam(args);
+    MutableIssue issue = getIssue(transientVars);
+
+    CustomField customField =
```

Comment:
You might want to add a bit more error handling here too. I think this block of code should be moved into a  method and shared with the create issue code.